mlflow/tracking/_tracking_service/utils.py

```diff
@@ -59,7 +59,7 @@ def set_tracking_uri(uri: Union[str, Path]) -> None:
                   "databricks://<profileName>".
                 - A :py:class:`pathlib.Path` instance
 
-    .. code-block:: python
+    .. test-code-block:: python
```

Comment:
We can create a good first issue and ask the community to update the existing code block.

Reply:
I haven't checked that, but I believe not too many.